book/website/reproducible-research/reviewing/reviewing-motivation.md

```diff
@@ -48,7 +54,7 @@ The more readable your code is, the easier it will be for other developers to re
 <a name="Style_enforcement"></a>
 ## Style enforcement
 
-Many projects enforce certain [code style guidelines](../../code_quality/code_quality#coding-style), be they widely-adopted standards (for example, [PEP8](https://www.python.org/dev/peps/pep-0008/), the [Google C++ style guide](https://google.github.io/styleguide/cppguide.html)) or more project-specific conventions. [Automated services](../../code_quality/code_quality#online-services-providing-software-quality-checks) provide a convenient way to enforce a coding style and start the discussion about code quality.
+Many projects enforce certain {ref}`code style guidelines<rr-code-quality>`, be they widely-adopted standards (for example, [PEP8](https://www.python.org/dev/peps/pep-0008/), the [Google C++ style guide](https://google.github.io/styleguide/cppguide.html)) or more project-specific conventions. [Automated services](../../code_quality/code_quality#online-services-providing-software-quality-checks) provide a convenient way to enforce a coding style and start the discussion about code quality.
```

Comment:
There is a typo, as _they widely-adopted standards_ should be _**the** widely-adopted standards_

Reply:
It is fixed.